data.json

```diff
@@ -638,6 +638,15 @@
     "dateAdded": "2023-10-11T14:16:05.000Z",
     "tags": ["clothing", "hacktoberfest", "stickers", "device"]
   },
+  {
+    "name": "Modtoberfest",
+    "difficulty": "medium",
+    "description": "Only pull requests made in the participating projects count towards your progress. You must submit at least 4 PRs to any of the participating repositories. You will get a T-Shirt, pins, and a tree will be planted!",
```

Comment:
Please update the description for it to look good on the website.

Reply:
i added new description